test/functional/p2p_blocksonly.py

```diff
@@ -59,7 +59,7 @@ def run_test(self):
 
         self.log.info('Check that txs from peers with relay-permission are not rejected and relayed to others')
         self.log.info("Restarting node 0 with relay permission and blocksonly")
-        self.restart_node(0, ["-persistmempool=0", "-whitelist=relay@127.0.0.1", "-blocksonly"])
+        self.restart_node(0, ["-persistmempool=0", "-whitelist=relay@127.0.0.1", "-blocksonly", '-deprecatedrpc=whitelisted'])
```

Comment:
This change doesn't seem to be needed, e.g. the test doesn't fail without it unless you also assert on the `whitelist` field (in which case, use double quotes like the other args in this line).

Reply:
The `whitelist` field has been asserted on before the rebase, so this is probably an unsolved silent conflict